content/docs/1.1.0/advanced-resources/deploy/revision_counter.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Revision Counter
+weight: 2
+---
+
+Revision Counter is a strong mechanism that Longhorn keeps tracking each replicas' updates. During replica creation, Longhorn will create a 'revision.counter' file, it's initial counter is 0. And for every write to the replica, the counter in 'revision.counter' file will be increased by 1. And Longhorn Engine will use these counters to make sure all replcias are consistent during start time. Also for salvage recovering, will use these counters to decide which replica has the latest update.
+
+Disable Revision Counter is an option to boost Longhorn performance which doesn't keep tracking every write on replicas, but lose the strong tracking for each replica. If some user prefers higher performance and doesn't need this extra tracking, this option can help in such case. During Longhorn Engine start time, it will skip checking the revision counter for all replicas. But Longhorn still support auto-salvage through the replica's head file stat. By default the revision counter is enabled.
```

Comment:
`and doesn't need this extra tracking`. Need some examples, e.g. `the users have a stable network infrastructure, with plenty of CPU resources.`